File: app/src/main/java/kr/co/sorizava/asrplayer/websocket/listener/WsStatusListener.java
package kr.co.sorizava.asrplayer.websocket.listener;

import okhttp3.Response;
import okio.ByteString;

public interface WsStatusListener {
  public void onConnectionStatusChanged(int status);
  public void onMessageSubtitle(String text, boolean isFinal);
  public void onMessageSubtitle(String text);
  public void resetSubtitleView();
}

File: app/src/main/java/kr/co/sorizava/asrplayerKt/websocket/listener/WsStatusListener.kt
package kr.co.sorizava.asrplayerKt.websocket.listener

open interface WsStatusListener {
    fun onConnectionStatusChanged(status: Int)
    fun onMessageSubtitle(text: String?, isFinal: Boolean)
    fun onMessageSubtitle(text: String?)
    fun resetSubtitleView()
}